tools-image/build-arm-image.sh

```diff
@@ -446,6 +446,8 @@ sync
 echo ">> Performing $model specific bits.."
 /arm/boards/$model.sh ${DRIVE}
 
+dmsetup remove $DRIVE
```

Comment:
in order to remove something with dmsetup we need to know what we allocate first. we need to replace losetup instances, so something among the lines of:

```
loop=$(losetup -f --show ...)

....
do the work
....

dmsetup remove /dev/mapper/...
kpartx -d $loop
losetup -d $loop
```